server/jobs/models/models.go

```diff
@@ -0,0 +1,25 @@
+package models
+
+type OutputBuffer struct {
```

Comment:
I think i commented about this in the last PR, I don't agree with the concept of using `models` as a filename or a package name because it just becomes a dumping ground of things eventually. Let's have names that make sense.

In this case the objects are also arbitrarily appended with `Context` so maybe an overall rethink on naming here is a good idea.

Reply:
I've moved the structs to project_command_output_handler.go since that's the only place they are being used and I've renamed `PullContext` and `JobContext` to `PullInfo` and `JobInfo` respectively. Lmk if you have any other naming suggestions 😅